scss/layout/_grid.scss

```diff
@@ -68,6 +69,16 @@
   }
 }
 
+.slab--primary .grid {
+  a {
```

Comment:
@xtine I'm fine with that having that change in a follow up, but we just need to not lose track of it.